### PR TITLE
[WIP] Handcuffs

### PR DESCRIPTION
**Pull request by ike709**
Adds handcuffs.

TODO:
- [x] The basics of handcuffs are functional.
- [ ] Figure out how to check if an entity has hands within the context of Shared.
- [ ] Player's mob should have handcuffs overlay on it.
- [ ] Optional: Cuff icon on the hand GUIs themselves?
- [ ] Cleanup

### CODE REVIEW

File: Content.Shared/GameObjects/Components/Interactable/CuffedComponent.cs [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+                if (isOwner) // TODO: do_after() once it exists
+                {
+                    Logger.InfoS("fuck", "Here 2");
+                    Timer.Spawn(TimeSpan.FromSeconds(breakoutTime), () => Uncuff(user, true), _cancellationTokenSource.Token);
+                    Logger.InfoS("fuck", "Here 3");
+                }
+                else
+                {
+                    _notifyManager.PopupMessage(user, Owner, $"{user.Name} starts removing the cuffs.");
+                    Logger.InfoS("fuck", "Here 4");
```
Comment (F77F) [inline Content.Shared/GameObjects/Components/Interactable/CuffedComponent.cs:99]: based

Comment (ike709) [reply Content.Shared/GameObjects/Components/Interactable/CuffedComponent.cs:99]: If it's stupid and it works it's not stupid.

Comment (F77F) [reply Content.Shared/GameObjects/Components/Interactable/CuffedComponent.cs:99]: exactly

**Comment by ike709** [comment]
I'm tired of working on this PR.

**Comment by F77F** [comment]
bruh momentt. what made you quit. i might take it up if its not that bad.

**Comment by kiVts** [comment]
Is this thing that mandatory?

**Comment by kiVts** [comment]
Actually a lot of work remaining, Soo, we will see handcuffs not that far

**Comment by nuke-makes-games** [comment]
I wouldn't mind taking a crack at it

**Comment by Zumorica** [comment]
Also comment on #1664 so we can assign you to the issue @nuke-makes-games

**Pull request closed without merging**